neo/public/stylesheets/main.css

```diff
@@ -119,7 +113,7 @@ search-bar {
 #powerby{
            position: absolute;
            bottom: 0%;
-           left: 71%;
+           left: 70%;
```

Comment:
I also thought that but if we zoom to 150% as it does when we connect to projector or when dev tools is popped up on it would look messy when we have it corner edge.Now there wont be any overlap of any  sorts.If you want I can change if it is ok to overlap when dev tools is opened or high zoom in browser is used.